### DISCUSSION TITLE
case sensitivity

### DISCUSSION THREAD

**Discussion by reymalahay** [mobile-dev-inc/Maestro#1534] [Q&A] [answered]
hi all,

when searching for text, how does `maestro` deal with the same word but with different case? for example, i i am looking for the word "Dog" on a page, and there are two instances, "dog" and "Dog". and what i want is the second occurrence, how do i deal with this without using regex?

please advise.

thanks in advance,
rey malahay

**Comment by Fishbowler** [comment, marked as answer]
Maestro is case insensitive.
You can look at the more complex selectors to say that you want the "dog" that's below "cat" or that's a childOf something, etc.
As a last resort, you can also `index` to indicate that it's the second one.
